src/common/dao/group/usergroup.go

```diff
@@ -84,6 +86,27 @@ func GetUserGroup(id int) (*models.UserGroup, error) {
 	return nil, nil
 }
 
+// GetHTTPGroupIDByGroupName ...
+func GetHTTPGroupIDByGroupName(groupName []string) ([]int, error) {
```

Comment:
1.  This is not restricted to HTTP group
2.  Is it possible we reuse `QueryUserGroup` ?  For example you can add a list of names to the input query parm.

I hope there's only one "ListGroup" func or there's only one fundamental "listGroup" func that is reused by different funcs.

Reply:
1. for future, we can make it general and add a groupType param
2. QueryUserGroup match groupName by exact name